src/mono/wasm/runtime-test.js

```diff
@@ -191,6 +195,8 @@ var Module = {
 		}
 
 		config.loaded_cb = function () {
+			FS.mkdir(working_dir);
+			FS.chdir(working_dir);
```

Comment:
This seems to be breaking some tests. Eg:
` System.IO.FileNotFoundException : Could not load file or assembly '/net5.0-Browser-debug/System.Runtime.Loader.Tests.dll' or one of its dependencies.` (from https://github.com/dotnet/runtime/pull/42020/checks?check_run_id=1092348599)

I would suggest checking what is the current working directory for tests. You can print that in this method to check too. Then it should give a better idea of what should be passed in as the `--working-dir`.